tests/test_integration/test_operations.py

```diff
@@ -0,0 +1,22 @@
+"""Test operations for the tus-storagehandler service."""
```

Comment:
I am not sure about this particular test, I don't understand what this test is supposed to test? Haven't reviewed this 👍

Reply:
I also don't understand it. Seems to me that the service is never called. You could put any URL in `requests.get()` and you'd always get the mock response. And since you check the result against the mock response, you're not really testing anything (except the correct function of the `mock` package.

Or am I missing something?

Mocking is certainly the right approach for unit tests. Also for integration tests when you want to make tests independent of third party services/tools. But here I think we really wanna see an end-to-end test that checks that we do get a response from a specific response from an endpoint.

For that:
- The service needs to be up and running.
- A dummy operation has to be implemented that gives a predictable response for a given status code ("hello world" for 200 or sth like that)
- The mock stuff has to be removed and the operation has to be _actually_ called against the running service.
- The _actual_ response should be check against the expected response (assert both the status code (e.g. 200) and the response value (e.g., "hello world").

> To avoid conflicts with the OpenAPI specification (which, depending on the FOCA configuration might throw an error for a simple "hello world response", you could perhaps implement somewhat less of a dummy controller that gives a response that is actually valid as per the specs, e.g., for `GET /service-info`, your controller could return a hard-coded, minimal, yet valid `ServiceInfo` response. In one of the next PRs, you could then replace the controller with a more involved implementation that pulls the service info from the database and maybe handles `401` and/or `403` responses as well, etc.